### PR TITLE
1.5.0 - enhancement #2 : add term and range searches on date with ui.…

### PR DESCRIPTION
**Pull request by oguennec**
Integration of angular-ui for bootstrap in order to display the Calendar (ui.bootstrap.datepicker) while enhancing date search capabilities with term and range search.

Different date formats for different date fields can be selected from the builder.

filterswithcalendar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,49 +66,64 @@ $scope.elasticBuilderData.fields = {
...
 Which represents the following Elasticsearch Query:
 ```json
 [
   {
-    "terms": {
-      "multi.selector": [
-        "AZ",
-        "CT"
-      ]
+    'and': [
```
Comment (olivierlesnicki) [inline README.md]: You should use double quotes (`"`) instead of single quotes

**Comment by oguennec** [comment]
@dncrews Hey Dan, I have submitted a PR for the date search enhancement. Here is a [running example](https://angular-elastic-builder.herokuapp.com/). Olivier

**Comment by oguennec** [comment]
@olivierlesnicki Thanks Olivier ! I have updated the json with double quotes inside README.md

**Comment by dncrews** [comment]
Thanks @oguennec. I'm going through it now, making some minor tweaks. Mostly it's linting stuff like spacing, etc, just keeping to the format the rest is in. Great work.

**Comment by dncrews** [comment]
Changed to 1.5.0 because of Semver. It's a feature addition, rather than a patch

**Comment by dncrews** [comment]
Merged. It just doesn't show it because I squashed your json change into your other one.

**Comment by dncrews** [comment]
Good work @oguennec

**Comment by oguennec** [comment]
Thanks @dncrews

**Comment by oguennec** [comment]
@dncrews Can you create a tag 1.5.0 ?

**Comment by dncrews** [comment]
Thanks @oguennec. I forgot to push my tags.

**Pull request closed without merging**